libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -155,6 +170,176 @@ static int32_t recvExact( MQTTTransportRecvFunc_t recvFunc,
     return totalBytesRecvd;
 }
 
+static bool publishPacketSize( const MQTTPublishInfo_t * pPublishInfo,
```

Comment:
Please check style guide for internal function naming. I think it is _xyzAbc()

Reply:
Sure.